content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -51,19 +51,17 @@ See the Google Kubernetes Engine (GKE) guide to [configuring cluster access for
     ```
     # The following parameters can be customized based on your needs.
 
+    gcloud config set project <your-gcp-project>
+
     CLUSTER_NAME="kubeflow-pipelines-standalone"
     ZONE="us-central1-a"
-    MACHINE_TYPE="n1-standard-2" # A machine with 2 CPUs and 7.50GB memory
-    SCOPES="storage-rw,cloud-platform" # These scopes are needed for running some pipeline samples
+    MACHINE_TYPE="n1-standard-2" # Minimum required resources
+    SCOPES="cloud-platform" # Allow talks with GCP APIs without additional settings
 
     gcloud container clusters create $CLUSTER_NAME \
         --zone $ZONE \
         --machine-type $MACHINE_TYPE \
-        --scopes $SCOPES \
-        --num-nodes 2 \
-        --max-nodes 5 \
-        --min-nodes 2 \
-        --enable-autoscaling
+        --scopes $SCOPES
```

Comment:
I'm curious why disable auto scaling?

Reply:
Default it to align with best GKE practice.
Actually the GKE auto-scale also can't work with KFP typed workload.

Reply:
Thanks, make sense to me